men_team_1/app/controllers/pairings_controller.rb

```diff
@@ -7,17 +7,41 @@ class PairingsController < ApplicationController
     def index
         @mentors = []
         @user.mentee_pairings.each do |mentor| 
```

Comment:
I would rename `mentor` to `pairing`, or something else that more closely represents the object (I noticed this because of how bizarre `mentor.mentor_id` looked below.

Reply:
Also, why can't this entire block of code just be:
```
@user.mentees
```
?